### PR TITLE
Grid and Rating added

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**
Creaded Grids and Rating
image
image
image
image

### CODE REVIEW

File: component/Grids/styles.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@import url('/component/NavBar-Vertical/styles.css');
+@import url('/styles/styles-global/root.css');
+@import url('/styles.css');
+
+.grid-two{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 0.5rem;
```
Comment (yaswanthmyneni) [inline component/Grids/styles.css]: use px units if it is less than 1rem. The remaining code looks good, you can merge it.

File: component/Grids/index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            <a class="nav-content"><i class="nav-logo fas fa-spa"></i></a>
+            <h2 class="nav-content nav-logo-txt">Roots</h2>
+        </div>
+        <div class="nav-child">
+            <a href="/index.html" class="nav-link nav-content">Home</a>
+            <a class="nav-link nav-content">Installation</a>
+            <a href="/component/Avatars/index.html" class="nav-link nav-content">Documentation</a>
+        </div>
+    </nav>
+    <div class="nav-body scroll">
```
Comment (sri-rishi) [inline component/Grids/index.html:28]: for sidebar use aside tags

File: component/Grids/index.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            </ul>
+        </div>
+    </div>
+
+    <!-- GRID BODY -->
+
+
+    <div class="component-body">
+        <h2 class="component-heading">Grids</h2>
+        <h5 class="component-content"></h5>
```
Comment (sri-rishi) [inline component/Grids/index.html]: remove empty tags

File: component/Grids/index.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        <!-- GRID 2 COLUMN -->
+        <h4 class="component-subheading"> Grid with two column </h4>
+        <div class="container">
+            <div class="grid-two">
+                <div class="grid-box"></div>
+                <div class="grid-box"></div>
+            </div>
+        </div>
+        <div class="iframe-container">
+            <code>
```
Comment (sri-rishi) [inline component/Grids/index.html:72]: Instead of div using only code tag and giving it styles will work

File: component/Grids/index.html [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                </iframe>
+            </code>
+        </div>
+    </div>
+
+
+
+
+
+
```
Comment (sri-rishi) [inline component/Grids/index.html]: remove extra spaces

**Review by sri-rishi — CHANGES_REQUESTED** [review]

File: component/Grids/index.html (hunk 4 above)
Comment (vanshitamoolchandani) [reply component/Grids/index.html:72]: It's not working that's why i had to give a container. Other than this all changes are done.

**Review by sri-rishi — APPROVED** [review]

**Pull request merged**